kf_lib_data_ingest/etl/configuration/target_api_config.py

```diff
@@ -1,338 +1,196 @@
 """
-A class which first validates and then stores transformation configuration
```

Comment:
I think we need to update this concept of Target API Config every where in the code and docs to Target API Ingest Plugin or Target Ingest Plugin.

Reply:
whyyyyyyyy?